lerna.json

```diff
@@ -6,7 +6,6 @@
 	},
 	"ignoreChanges": [
 		"**/benchmark/*.js",
-		"**/CHANGELOG.md",
```

Comment:
This change should be reverted after we fix the current packages issue.

Reply:
Created #10823 to revert this change